packages/node/src/stellar/api.stellar.ts

```diff
@@ -311,6 +293,14 @@ export class StellarApi implements ApiWrapper<StellarBlockWrapper> {
       try {
         eventsForSequence = await this.getAndWrapEvents(sequence);
       } catch (e) {
+        if (e.message === 'start is after newest ledger') {
+          const latestLedger = (await this.sorobanClient.getLatestLedger())
+            .sequence;
+          throw new Error(`The requested events for ledger number ${sequence} is not available on the current soroban node.
+                This is because you're trying to access a ledger that is earlier than the latest ledger number ${latestLedger} stored in this node.
```

Comment:
Should this be
```suggestion
                This is because you're trying to access a ledger that is after the latest ledger number ${latestLedger} stored in this node.
```